app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
```

Comment:
Consider providing a docstring for the class and methods, it could improve code maintainability.
Add below initialization something like that: 
```suggestion
    def __init__(self, name: str) -> None:
         """
         Initialize a Cafe instance with a given name.
         """
 ```